infra/discovery/src/lib/search.js

```diff
@@ -178,16 +189,16 @@ class Listing {
    */
   static async search(query, sort, order, filters, numberOfItems, offset) {
     const currencies = [
```

Comment:
Agreed we need to do some refactoring. There are currencies defined in many different places. DApp, GraphqQL, Discovery, Bridge...
Now I'm hesitant adding a dependency to the discovery server on graphQL.
I'm thinking we should perhaps have a new package, like "constants", that would contain all those definitions.
For the short-term, I've added a Note in the GraphQL code so that we remember to update the other constants if making change to that list there.
I'm also going to create a separate issue for adding a new "constants" package.